includes/class.xprofile.php

```diff
@@ -320,6 +370,14 @@ protected function _scan_needed() {
                         if ( !$is_registered ) {
                             return true;
                         }
+                        $meta = bp_xprofile_get_meta( $field->id, 'field', 'alternate_name', true );
+                        if ( ! empty( $meta ) ) {
```

Comment:
```suggestion
                        if ( $meta ) {
```

Sorry to be insistent, but it looks so much simpler to me :)

Reply:
It's just habit ... and in this particular case, it was already all over the place.

I'll keep an eye on this in the future.